browser/css/calculator.css

```diff
@@ -0,0 +1,97 @@
+@font-face {
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  font-size: 100%;
+  box-sizing: border-box;
+}
+
+.page {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+}
+
+.calculator {
+  height: 320px;
+  width: 233px;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  border: solid grey 1px;
+  box-shadow: 15px 15px 40px #8c8c8c;
+}
+
+.calculator-output-row {
+  background-color: #999999;
+  border-top-left-radius: 10px;
+  border-top-right-radius: 10px;
+  height: 25%;
+  padding: 10px;
+  color: white;
+  font-size: 200%;
+  font-weight: lighter;
+  display: flex;
+  align-items: center;
+  flex-direction: row-reverse;
+
+}
+
+.calculator-row {
+  display: flex;
+  height: 20%;
```

Comment:
I would avoid this relative size strategy. The calculator does not need to scale. It can be a fixed pixel size. I'd go with a static pixel based approach rather than relative percentages.